apps/console/src/extensions/configs/models/server-configuration.ts

```diff
@@ -53,9 +53,28 @@ export interface ServerConfigurationConfig {
         setPasswordHistoryEnabled: (state: boolean) => void,
         t: TFunction<"translation", undefined>
     ) => ReactElement;
+
+    usePasswordExpiry: () => RequestResultInterface<GovernanceConnectorInterface, RequestErrorInterface>;
+    processPasswordExpiryInitialValues: (
+        initialValues: ValidationFormInterface,
+        passwordExpiry: GovernanceConnectorInterface,
+        setPasswordExpiryEnabled: (state: boolean) => void
+    ) => PasswordExpiryInterface;
+    processPasswordExpirySubmitData: (data: ValidationFormInterface) => Promise<any>;
+    passwordExpiryComponent: (
+        componentId: string,
+        passwordExpiryEnabled: boolean,
+        setPasswordExpiryEnabled: (state: boolean) => void,
+        t: TFunction<"translation", undefined>
+    ) => ReactElement;
 }
 
 export interface PasswordHistoryCountInterface extends ValidationFormInterface {
-    passwordHistoryCount: number;
+    passwordHistoryCount: number | string;
```

Comment:
Do we need to add `string` here? We are not changing the `passwordHistoryCount` right?

Reply:
Addressed